src/libs/Navigation/AppNavigator/modalCardStyleInterpolator.js

```diff
@@ -2,7 +2,8 @@ import {Animated} from 'react-native';
 import variables from '../../../styles/variables';
 
 export default (
-    isSmallScreen,
+    isSmallScreenWidth,
+    isfullScreenModal,
```

Comment:
Are we still using this? I don't see where we are passing this in anywhere

Reply:
Yes in AuthScreens.js

Reply:
Ah ok I see now, for some reason I thought this was being destructured and expected to find another reference somewhere.